shell/platform/common/accessibility_bridge.h

```diff
@@ -31,85 +31,17 @@ namespace flutter {
 /// FlutterPlatformNodeDelegate to wrap each AXNode in order to provide
 /// an accessibility tree in the native format.
 ///
-/// This class takes in a AccessibilityBridgeDelegate instance and is in charge
-/// of its lifecycle. The delegate are used to handle the accessibility events
-/// and actions.
-///
-/// To use this class, you must provide your own implementation of
-/// FlutterPlatformNodeDelegate and AccessibilityBridgeDelegate.
+/// To use this class, you must implement this class and provide your own
```

Comment:
```suggestion
/// To use this class, one must subclass this class and provide their own
```

Reply:
should it also mention the onaccessibilityevent?

Reply:
I'm not sure, can you help me write it?